torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -551,8 +551,8 @@ def decompress(fut):
         )
 
         if state.use_error_feedback:
-            # memoize the local errors.
-            state.error_dict[bucket_index] = input_tensor_cp - input_tensor
+            # Memorize the local errors.
+            state.error_dict[bucket_index].copy_(input_tensor_cp - input_tensor)
```

Comment:
Is there a reason for this change? IIUC a copy is more extensive, as `a = b` for tensors just makes them point to the same underlying storage, but this actually results in a full data copy?

Reply:
Thanks! I thought it could save some peak memory footprint. Let me double confirm if there is a performance regression.

Reply:
You are right. There is a performance loss due to `copy_`. Changed it back to assignment. Good catch!